readthedocs/settings/base.py

```diff
@@ -333,132 +333,39 @@ def INSTALLED_APPS(self):  # noqa
     SILENCED_SYSTEM_CHECKS = ['fields.W342']
 
     # Logging
-    LOG_FORMAT = "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s"
+    LOG_FORMAT = '%(name)s[%(process)d]: %(levelname)s %(message)s [%(name)s:%(lineno)s]'
     LOGGING = {
         'version': 1,
         'disable_existing_loggers': True,
         'formatters': {
-            'standard': {
+            'default': {
                 'format': LOG_FORMAT,
                 'datefmt': "%d/%b/%Y %H:%M:%S"
```

Comment:
I'd be interested in this, but the format of DATETIME_FORMAT doesn't seem to be compatible with the logging datetime format.